Sources/BijectiveDictionary/Documentation.docc/Creating a BijectiveDictionary.md

```diff
@@ -0,0 +1,60 @@
+# Creating a BijectiveDictionary
+
+Learn the many different ways to initialize a `BijectiveDictionary`.
+
+## Creating an Empty `BijectiveDictionary`
+An empty dictionary can be created using ``BijectiveDictionary/init()``
+Because `BijectiveDictionary` is generic, you will also need to somewhere declare your `Left` and `Right` value types. 
+
+```swift
+let bDict = BijectiveDictionary<String, Int>()
+```
+
+### Reserving Capacity
+An empty dictionary can also be created with a minimum capacity using ``BijectiveDictionary/init(minimumCapacity:)``. Remember that, (just like a standard `Dictionary`) when a `BijectiveDictionary` exceeds it's capacity, it must reallocate its storage buffer under the hood. This reallocation is automatic and you do not need to remember to do it, but it is not free. It is good to avoid this work, if you know in advance that you will have a large number of items. 
+
+```swift
+let bDict = BijectiveDictionary<String, Int>(minimumCapacity: 10_000)
+```
+
+## Initialize with Values
+### Create a `BijectiveDictionary` from a Literal
+See: ``BijectiveDictionary/init(dictionaryLiteral:)``
+
+``BijectiveDictionary`` conforms to `ExpressibleByDictionaryLiteral` and therefore can be initialized with common `Dictionary` syntax. (Note, this creates a `BijectiveDictionary` directly, and does not create any `Dictionary`.) In practice, this is one of the easiest ways to create a `BijectiveDictionary`.  
+```swift
+let bDict: BijectiveDictionary = ["A": 1, "B": 2, "C": 3]
+```
+
+
+### Initialize with a Sequence of Tuples
+See ``BijectiveDictionary/init(uniqueLeftRightPairs:)``
```

Comment:
This initializer can potentially result in a fatal error. I am going to update the docs to make this more clear (issue #17), but maybe it is worth mentioning here as well.